utils/alsoft-config/mainwindow.cpp

```diff
@@ -882,7 +882,7 @@ void MainWindow::loadConfig(const QString &fname)
             }
             else if(backend.size() > 1)
             {
-                QStringRef backendref{backend.rightRef(backend.size()-1)};
+                QStringView backendref = backend.right(backend.size()-1);
```

Comment:
`right` seems to return a new `QString`, which would be assigning it to a non-owning QStringView and leave it dangling. I think this needs to be

```c++
auto backendref = QStringView{backend}.right(backend.size()-1);
```

Reply:
thanks!